README.md

```diff
@@ -1 +1,252 @@
 # Coreutils-EYAD_HUSSEIN
+
+Coreutils-EYAD_HUSSEIN is a collection of Unix-like command line utilities implemented in Go.
+
+## Project Structure
+
+```bash
+.
+├── bin
+├── pkg
+├── README.md
+└── src
+    └── Coreutils-EYAD_HUSSEIN
+        ├── cmd
+        │   ├── cat
+        │   │   └── main.go
+        │   ├── echo
+        │   │   └── main.go
+        │   ├── env
+        │   │   └── main.go
+        │   ├── false
+        │   │   └── main.go
+        │   ├── head
+        │   │   └── main.go
+        │   ├── tail
+        │   │   └── main.go
+        │   ├── tree
+        │   │   └── main.go
+        │   ├── true
+        │   │   └── main.go
+        │   ├── wc
+        │   │   └── main.go
+        │   └── yes
+        │       └── main.go
+        ├── go.mod
+        └── pkg
+            ├── cat.go
+            ├── echo.go
+            ├── env.go
+            ├── false.go
+            ├── head.go
+            ├── tail.go
+            ├── tree.go
+            ├── true.go
+            ├── wc.go
+            └── yes.go
+```
+- **cmd/**: Holds commands directories and files.
+- **pkg/**: Holds package files.
+- **src/Coreutils-EYAD_HUSSEIN/**: Source directory for the project.
+
+## Commands Implemented
```

Comment:
please provide a description to every flag

Reply:
Done